packages/ckeditor5-find-and-replace/src/ui/findandreplaceformview.js

```diff
@@ -47,190 +47,133 @@ export default class FindAndReplaceFormView extends View {
 		 * @observable
 		 * @member {Boolean} #isSearching
 		 */
-		this.set( 'isSearching' );
-		this.set( 'searchText', '' );
-		this.set( 'replaceText', '' );
+		this.set( 'isSearching', false );
 
 		/**
-		 * Stores the number of matched search results.
+		 * Searched text value.
 		 *
 		 * @readonly
 		 * @observable
-		 * @member {Number} #matchCount
+		 * @member {String} #searchText
 		 */
-		this.set( 'matchCount', null );
+		this.set( 'searchText', '' );
 
 		/**
-		 * The offset of currently highlighted search result in {@link #matchCount matched results}.
-		 *
-		 * @readonly
-		 * @observable
-		 * @member {Number|null} #highlightOffset
-		 */
-		this.set( 'highlightOffset', null );
+		  * Replace text value.
+		  *
+		  * @readonly
+		  * @observable
+		  * @member {String} #replaceText
+		  */
+		this.set( 'replaceText', '' );
 
 		/**
-		 * Whether the search results counter should be visible.
+		 * Indicates if searched text has been changed in form.
```

Comment:
This property should not be limited to sniff changes in searched text. It should also sniff changes in match case, whole words options.